examples/rabbitmq-consumer/rabbitmq-consumer.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on asynchronously consuming messages from a RabbitMQ queue using Ballerina.
```

Comment:
Suggested edit: BBE on consuming messages from a RabbitMQ queue in an asynchronous manner using Ballerina.